### PR TITLE
Add osu!trivium Quiz articles and link to them in `People/Users_with_unique_titles`

### PR DESCRIPTION
**Pull request by Walavouchey**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

The banner for the 2021 quiz (`Contests/osu!trivium_Quiz/2021/img/banner.png`) is taken from its [news post](https://osu.ppy.sh/home/news/2021-11-10-osu!trivium-quiz-2021), but it does incorrectly read "osu! trivium quiz" when it should be "osu!trivium Quiz", but that can hopefully be fixed later.

### CODE REVIEW

File: wiki/Contests/osu!trivium_Quiz/2020/en.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
+---
+tags:
+  - trivium
+  - quiz
+---
+
+# osu!trivium Quiz 2020
+
+The **osu!trivium Quiz 2020** was a free-for-all osu! trivia quiz contest hosted by [mangomizer](https://osu.ppy.sh/users/1893718). It was the first osu!trivium Quiz, the world's hardest quiz on osu!.
```
Comment (eveningsteps) [inline wiki/Contests/osu!trivium_Quiz/2020/en.md:9]:
```suggestion
The **osu!trivium Quiz 2020** was a free-for-all osu! trivia quiz contest hosted by ![][flag_GB] [mangomizer](https://osu.ppy.sh/users/1893718). It was the first osu!trivium Quiz, the world's hardest quiz on osu!.
```

File: wiki/Contests/osu!trivium_Quiz/2020/en.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+| Insane | [31](https://youtu.be/KIgdjSleasg?t=4422), [32](https://youtu.be/KIgdjSleasg?t=4600), [33](https://youtu.be/KIgdjSleasg?t=4710), [34](https://youtu.be/KIgdjSleasg?t=4858), [35](https://youtu.be/KIgdjSleasg?t=4983), [36](https://youtu.be/KIgdjSleasg?t=5149), [37](https://youtu.be/KIgdjSleasg?t=5238), [38](https://youtu.be/KIgdjSleasg?t=5626), [39](https://youtu.be/KIgdjSleasg?t=5720), [40](https://youtu.be/KIgdjSleasg?t=5872) |
+| Extra | [41](https://youtu.be/KIgdjSleasg?t=6080), [42](https://youtu.be/KIgdjSleasg?t=6202), [43](https://youtu.be/KIgdjSleasg?t=6347), [44](https://youtu.be/KIgdjSleasg?t=6495), [45](https://youtu.be/KIgdjSleasg?t=6762), [46](https://youtu.be/KIgdjSleasg?t=6928), [47](https://youtu.be/KIgdjSleasg?t=7040), [48](https://youtu.be/KIgdjSleasg?t=7172), [49](https://youtu.be/KIgdjSleasg?t=7400), [50](https://youtu.be/KIgdjSleasg?t=7687) |
+
+## Ruleset
+
+- You cannot ask [mangomizer](https://osu.ppy.sh/users/1893718) for information, hints, or answers.
```
Comment (eveningsteps) [inline wiki/Contests/osu!trivium_Quiz/2020/en.md:71]:
```suggestion
- You cannot ask ![][flag_GB] [mangomizer](https://osu.ppy.sh/users/1893718) for information, hints, or answers.
```

File: wiki/Contests/osu!trivium_Quiz/2021/en.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+tags:
+  - trivium
+  - quiz
+---
+
+# osu!trivium Quiz 2021
+
+![osu!trivium Quiz 2021 banner](img/banner.png)
+
+The **osu!trivium Quiz 2021** was a free-for-all osu! trivia quiz contest hosted by [mangomizer](https://osu.ppy.sh/users/1893718). It was the second osu!trivium Quiz, the world's hardest quiz on osu!.
```
Comment (eveningsteps) [inline wiki/Contests/osu!trivium_Quiz/2021/en.md:11]:
```suggestion
The **osu!trivium Quiz 2021** was a free-for-all osu! trivia quiz contest hosted by ![][flag_GB] [mangomizer](https://osu.ppy.sh/users/1893718). It was the second osu!trivium Quiz, the world's hardest quiz on osu!.
```

File: wiki/Contests/osu!trivium_Quiz/2021/en.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+| Extra | [41](https://www.twitch.tv/videos/1237832029?t=02h04m03s), [42](https://www.twitch.tv/videos/1237832029?t=02h07m48s), [43](https://www.twitch.tv/videos/1237832029?t=02h10m54s), [44](https://www.twitch.tv/videos/1237832029?t=02h14m31s), [45](https://www.twitch.tv/videos/1237832029?t=02h17m16s), [46](https://www.twitch.tv/videos/1237832029?t=02h19m01s), [47](https://www.twitch.tv/videos/1237832029?t=02h25m37s), [48](https://www.twitch.tv/videos/1237832029?t=02h28m53s), [49](https://www.twitch.tv/videos/1237832029?t=02h33m15s), [50](https://www.twitch.tv/videos/1237832029?t=02h37m11s) |
+
+## Ruleset
+
+- You cannot ask [mangomizer](https://osu.ppy.sh/users/1893718) or [peppy](https://osu.ppy.sh/users/2) for information, hints, or answers.
```
Comment (eveningsteps) [inline wiki/Contests/osu!trivium_Quiz/2021/en.md:75]:
```suggestion
- You cannot ask ![][flag_GB] [mangomizer](https://osu.ppy.sh/users/1893718) or ![][flag_AU] [peppy](https://osu.ppy.sh/users/2) for information, hints, or answers.
```

**Review by eveningsteps — COMMENTED** [review]
looks good to me otherwise

File: wiki/Contests/osu!trivium_Quiz/2020/en.md (hunk 2 above)
Comment (TicClick) [reply wiki/Contests/osu!trivium_Quiz/2020/en.md:71]: I don't think the flag is necessary here (same goes for other ruleset suggestions of yours)

**Review by TicClick — APPROVED** [review]

**Pull request merged**